home/index.html

```diff
@@ -134,19 +138,19 @@ <h1>Contact Us</h1>
           <form method="post">
             <label for="name">Name </label><input class="field" type="text" id="name" name="name"><br>
             <label for="email">E-mail </label><input class="field" type="email" id="email" name="email"><br>
-            <textarea class="message field" name="message"></textarea><br>
+            <textarea class="message field" name="message" placeholder="Enter your message here !"></textarea><br>
             <input class="call-to-action" type="submit" value="Submit">
           </form>
         </div>
     </section>
-    <section class="one is-clone">
+    <!-- <section class="one is-clone">
       <div class="content">
         <h1>Who we are</h1>
         <p><b>Codezilla</b> is a community for those who see things from a different perspective - the Contributors. We are a Mozilla verified Community and a Campus Club of SRM Ramapuram that provides a platform for community, code, and collaboration.</p>
         <a href="#three"><button class="call-to-action">Event Sign Up</button></a>
         <a href="#five"><button class="call-to-action">Contact</button></a>
       </div>
-    </section>
+    </section> -->

```

Comment:
I'm afraid that this is a breaking change because:
- This removes the infinite smooth scroll function

Reply:
Okay reverting that right away !

Reply:
Thank you for reverting this @KlausMikhaelson, but I belive it would be better to start over with a new PR